### PR TITLE
added SPID to queryEditorInput title

### PR DESCRIPTION
**Pull request by smartguest**
Fixes: #24806

SPID to Query Editor Title

### CODE REVIEW

File: src/sql/workbench/common/editor/query/queryEditorInput.ts [hunk 1]
```diff
@@ -284,6 +285,11 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 			fullTitle += ` (${profile.userName || profile.authenticationType})`;
 
 			fullTitle += additionalOptions;
+
+			if (info && info.serverConnectionId) {
```
Comment (kisantia) [inline src/sql/workbench/common/editor/query/queryEditorInput.ts:289]: nit: can simplify to `if (info?.serverConnectionId)`

**Review by kisantia — APPROVED** [review]

File: src/sql/workbench/common/editor/query/queryEditorInput.ts [hunk 2]
```diff
@@ -284,6 +285,11 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
...
+				// Add server info to query editor in case if it's available
+				fullTitle += ` ${localize('queryEditorInput.serverConnectionId', "Server Connection Process ID: {0}", info.serverConnectionId)}`
```
Comment (cheenamalhotra) [inline src/sql/workbench/common/editor/query/queryEditorInput.ts:291]: Can we simplify it to how SSMS shows - also add it in the beginning of the title?

**Comment by smartguest** [comment]
Here are two possible choices for how the SPID should be added in front, the second one is the current option: 
Choice1SPIDTitle
Choice2SPIDTitle

Let me know which one is better for users.

**Comment by cheenamalhotra** [comment]
Both look good to me as they provide visibility to user from the title without need to hover as SSMS.

We can go for the first one to keep it close to server information.
image

**Review by cheenamalhotra — APPROVED** [review]

**Pull request merged**